libs/brig-types/src/Brig/Types/Intra.hs

```diff
@@ -24,19 +24,22 @@ data AccountStatus
     = Active
     | Suspended
     | Deleted
+    | Ephemeral
```

Comment:
I would have added the ttl to the 'Ephemeral' constructor intsead of adding an extra field to the status-carrying types.

- This way an ephemeral user can never have 'Nothing' in its ttl.
- Makes 'toUsers', 'toUserAccount' easier: no need to pass a ttl if it's not needed.
- Complicates storage in cassandra a little.
- Not sure if it works at all.  Or if it's worth it.

Just an idea.  (-:

Reply:
This might be an idea for a separate PR.